src/Main.elm

```diff
@@ -96,10 +98,18 @@ view model =
             Theme.PageTemplate.view model
                 { title = SiteTitle, content = Page.Index.view model }
 
-        CaseStudy ->
+        CaseStudyIndex ->
             Theme.PageTemplate.view model
                 { title = CaseStudyTitle, content = Page.CaseStudy.view model }
 
-        Resource ->
+        CaseStudy _ ->
```

Comment:
I think we should use the param to render individual title per page. We can use a slug to title function.

Reply:
Same for resource.